### PR TITLE
Prevent dropdown to open if empty on refresh

### PR DESCRIPTION
**Pull request by tleon**
prevent empty dropdown to open on refresh and first loading

### CODE REVIEW

File: templates/github/pull-request/filtersNotifications.html.twig [hunk 1]
```diff
@@ -13,7 +13,7 @@
             </button>
         </div>
 
-        <div id="collapse-notifs-{{ repository|slugify }}" class="collapse show"
+        <div id="collapse-notifs-{{ repository|slugify }}" class="{{ unreadNotificationsCount[repository] > 0 ? 'collapse show' : 'collapse' }}"
```
Comment (babeuloula) [inline templates/github/pull-request/filtersNotifications.html.twig]:
```suggestion
        <div id="collapse-notifs-{{ repository|slugify }}" class="collapse {{ unreadNotificationsCount[repository] > 0 ? 'show' : '' }}"
```

File: templates/github/pull-request/filters.html.twig [hunk 2]
```diff
@@ -27,6 +27,13 @@
 
     <script type="text/javascript">
         jQuery(function ($) {
+
```
Comment (babeuloula) [inline templates/github/pull-request/filters.html.twig]:
```suggestion
```

File: templates/github/pull-request/filtersPullRequests.html.twig [hunk 3]
```diff
@@ -8,7 +8,7 @@
             </button>
         </div>
 
-        <div id="collapse-pr-{{ repository|slugify }}" class="collapse show"
+        <div id="collapse-pr-{{ repository|slugify }}" class="{{ sortedPullRequests.count() > 0 ? 'collapse show' : 'collapse' }}"
```
Comment (babeuloula) [inline templates/github/pull-request/filtersPullRequests.html.twig]:
```suggestion
        <div id="collapse-pr-{{ repository|slugify }}" class="collapse {{ sortedPullRequests.count() > 0 ? 'show' : '' }}"
```

File: templates/github/pull-request/filters.html.twig [hunk 4]
```diff
@@ -27,6 +27,12 @@
 
     <script type="text/javascript">
         jQuery(function ($) {
+            window.hideEmpty = function () {
+                $('[data-empty="true"]').map(function (key, value) {
+                    $($(value).parents('.collapse').get(0)).removeClass('show');
+                });
+            };
+
```
Comment (jeremy-wizaplace) [inline templates/github/pull-request/filters.html.twig]: With your last update, you JS is useless ^_^

**Pull request merged**